webapp/certification/views.py

```diff
@@ -173,13 +175,14 @@ def certification_home():
         query = request.args.get("text", default=None, type=str)
         limit = request.args.get("limit", default=20, type=int)
         offset = request.args.get("offset", default=0, type=int)
+        filters = request.args.get("filters", default=False, type=bool)
 
         forms = (
             ",".join(request.args.getlist("form"))
             if request.args.getlist("form")
             else None
         )
-        if "Models" in forms:
+        if forms and "Models" in forms:
             forms = ",".join(
```

Comment:
The API does a search for all forms when nothing is passed, so in this case you can just leave it as None.

You can see an example by leaving out `category` as a query parameter:
https://certification.staging.canonical.com/api/v1/certifiedmodels/?format=json

Reply:
This has been resolved - we need to pass this so the check can be done in the front end to default tick each category on the search results page.